src/cloud_init.yml

```
@@ -0,0 +1,16 @@
+---
+# Ensure that cloud-config is enabled and enable the others while we're at it.
+- hosts: all
+  name: Enable all cloud-init services
+  become: yes
+  become_method: sudo
+  tasks:
+    - name: Enable all cloud-init services
+      ansible.builtin.service:
+        name: "{{ item }}"
+        enabled: yes
+      loop:
+        - cloud-init-local
+        - cloud-init
+        - cloud-config
+        - cloud-final
```

We prefer to sort things like this. If the current order is required then it should be documented. I see this appears to be an inherited thing from cribbing other Packer projects so as long so we should update those projects based on the resolution of this conversation.

```suggestion
        - cloud-config
        - cloud-final
        - cloud-init
        - cloud-init-local
```

---

Yes, this list of items should be sorted here and elsewhere.